frontend/src/components/TitleBar/Settings/SettingsButton.jsx

```diff
@@ -15,7 +15,7 @@ export default function SettingsButton() {
   return (
     <div style={{ display: 'inline-block' }}>
       <button type="submit" onClick={displayModal}>
-        <Settings />
+        <Settings className={styles.SettingsButton} />
```

Comment:
@meganyin13 I changed this line. The css module names are case-sensitive. I think this pull request is ready to go.